db/migrations/20201229214901-create-actividades.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable(
+      'Actividads',
+      {
+        id: {
+          allowNull: false,
+          autoIncrement: true,
+          primaryKey: true,
+          type: Sequelize.INTEGER,
+        },
+        espacioId: {
+          allowNull: false,
+          type: Sequelize.INTEGER,
+          references: {
+            model: 'Espacios',
+            key: 'id',
+          },
+        },
+        nombre: {
+          allowNull: false,
+          type: Sequelize.STRING,
+        },
+        fechaHoraInicio: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        fechaHoraFin: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        responsable: {
+          allowNull: false,
+          type: Sequelize.STRING,
+        },
+        dniResponsable: {
+          allowNull: false,
+          type: Sequelize.INTEGER,
+        },
+        tipoResponsable: {
+          allowNull: false,
+          type: Sequelize.STRING,
+        },
+        estado: {
+          defaultValue: true,
+          type: Sequelize.BOOLEAN,
+        },
+        requiereControl: {
+          defaultValue: false,
+          type: Sequelize.BOOLEAN,
+        },
+        createdAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+        updatedAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+        },
+      },
+      {
+        esquema: 'public',
+        tableName: 'Actividads',
+      },
+      { freezeTableName: true }
```

Comment:
Nunca había visto esto tampoco, ¿para qué sirve?

Reply:
Las primeras dos sirven para indicar en que esquema de la bdd se guarda la tabla, y la segunda es el nombre. Lo pusimos porque tuvimos problemas cuando haciamos la migración nos cambiaba el nombre y nos ponia el nombre del modelo: actividad + una S, osea, 'actividads', probamos eso y el otro parámetro "freezeTableName" que encontramos para que no cambie de nombre la tabla, pero no funcionó, terminamos llamando a la tabla 'actividads', este problema lo habiamos tenido con sequelize también en Persistencia y terminamos resolviendo lo mismo porque no parece haber otra forma.

Reply:
Esquema en español dudo que funcione. Además, según la doc dd Sequelize esta configuración va en el Modelo, no acá.

```js
class Bar extends Model {}
Bar.init({ /* bla */ }, {
  // The name of the model. The model will be stored in `sequelize.models` under this name.
  // This defaults to class name i.e. Bar in this case. This will control name of auto-generated
  // foreignKey and association naming
  modelName: 'bar',

  // don't add the timestamp attributes (updatedAt, createdAt)
  timestamps: false,

  // don't delete database entries but set the newly added attribute deletedAt
  // to the current date (when deletion was done). paranoid will only work if
  // timestamps are enabled
  paranoid: true,

  // Will automatically set field option for all attributes to snake cased name.
  // Does not override attribute with field option already defined
  underscored: true,

  // disable the modification of table names; By default, sequelize will automatically
  // transform all passed model names (first parameter of define) into plural.
  // if you don't want that, set the following
  freezeTableName: true,

  // define the table's name
  tableName: 'my_very_custom_table_name',

  // Enable optimistic locking.  When enabled, sequelize will add a version count attribute
  // to the model and throw an OptimisticLockingError error when stale instances are saved.
  // Set to true or a string with the attribute name you want to use to enable.
  version: true,

  // Sequelize instance
  sequelize,
})
```
(sacado de https://sequelize.org/v5/manual/models-definition.html, sección **Configuration**)

Prueben con definir `tableName`, eso debería funcionar.

Reply:
En resumen: el problema no está en la migración sino en cómo Sequelize espera que se llame la tabla. En las migraciones nunca van a configurar nada de esto porque lo único que hacen es ejecutar código SQL.

Reply:
Última, también veo que aparentemente también pueden poner esto en la config del modelo:

```js
{ 
  name: { singular: 'Actividad', plural: 'Actividades' }
}
```

Porfa pruebenlo. Van a tener que deshacer la migración y volver a crear la tabla con nombre `Actividades`.